msal/src/test/java/android/text/TextUtils.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2006 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Workaround for:
+// java.lang.RuntimeException: Method isEmpty in android.text.TextUtils not mocked.
+// See http://g.co/androidstudio/not-mocked for details.
+package android.text;
+
+public class TextUtils {
```

Comment:
Isn't this available in android?

Reply:
ah... Alternatively, we could use our own StringUtil.

Reply:
Which is a little different.  our isEmtpy is a non-performant version of isBlank(), since we trim whitespace.  We should change it to use a static pattern.